### DISCUSSION TITLE
My Step 2 of this courseis unable to complete please help me it i have done every step right.

### DISCUSSION THREAD

**Discussion by Amanlath1** [skills/.github#605] [Introduction to GitHub]
It is not going to step 3.

**Comment by scordle** [comment]
I am having the same issue, Here is [my repo](https://github.com/scordle/skills-introduction-to-github/tree/my-first-branch)

I attempted step 2 (add a file, first commit) and cannot get the Actions to take the next step. Any help is appreciated.
